packages/documentation/src/stories/foundations/icons/icon.stories.ts

```diff
@@ -178,9 +178,9 @@ export const Flip: Story = {
 };
 
 const scaleVariants = [
-  { class: 'bg-info', scale: 0.5 },
-  { class: 'bg-info' },
-  { class: 'bg-info', scale: 1.5 },
+  { class: 'my-icon', scale: 0.5 },
```

Comment:
It would be good to add a decorator so that `.my-icon` class is not showing in the code snippet.